google/cloud/internal/async_polling_loop_test.cc

```diff
@@ -118,20 +121,78 @@ TEST(AsyncPollingLoopTest, ImmediateCancel) {
             make_status_or(std::chrono::system_clock::now()));
       });
   CompletionQueue cq(mock_cq);
+  auto current =
+      MakeImmutableOptions(Options{}.set<StringOption>("ImmediateCancel"));
```

Comment:
You might want to consider `testing::UnitTest::GetInstance()->current_test_info()->name()`.